### ISSUE TITLE
Adaption for the german language

### ISSUE THREAD

**Issue by JulianGerhard21** [PolyAI-LDN/polyai-models#24]
Hi guys,

as I already mentioned in my email, I'd like to adapt your model for the german language. Would you be interested in this? I really like the approach and I would have the ressources to provide the training data. In detail I would be interested in the advantage of ConveRT over a domain-finetuned BERT in our use cases.

Kind regards
Julian

**Comment by matthen** [comment]
Hi Julian,
Thanks for your interest! What sort of training data do you have? (how much of it, is it input-response..?)
We have some approaches for multilingual but would like to explore it further. Feel free to get in touch using matt@matthen.com
